portals/knowledge-management/src/App.vue

```diff
@@ -362,7 +362,7 @@
 <template>
     <div
         class="font-sans bg-gray-50 min-h-screen"
-        :class="{ 'mx-0 md:mx-10 lg:mx-24': portalLayout === 'fixed' }"
+        :class="{ 'mx-0 md:mx-10 lg:mx-40': portalLayout === 'fixed' }"
```

Comment:
@ketan-canyon Let's make sure this is setting a max width instead of the margin. We can remove the other margins as well.